threes.py

```diff
@@ -106,20 +111,20 @@ def play_game_interactive():
 
     while True:
         m, tileset, valid = game.send(move)
-        print to_val(m)
+        print(to_val(m))
 
         if not valid:
-            print "Game over."
-            print "Your score is", to_score(m).sum()
+            print("Game over.")
+            print("Your score is", to_score(m).sum())
```

Comment:
Please add `from __future__ import print_function` to maintain Py2 compatibility.